### PR TITLE
Marine shields

### PR DESCRIPTION
**Pull request by InterroLouis**
## About The Pull Request

Marine shields

## Why It's Good For The Game

Marine shields

## Changelog
:cl:
add: Marine shields exist
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -1. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by Hughgent** [comment]
Were you considering making this available in vendors or anywhere other than admin spawn?

**Comment by Pariah919** [comment]
when you add the shields to req make them cost 45 initially and add 1 to the operational vendor

File: code/game/objects/items/weapons/shields.dm [hunk 1]
```diff
@@ -125,6 +125,15 @@
 		return TRUE
 	return ..()
 
+/obj/item/weapon/shield/riot/marine
+	name = "TL-172 Defensive Shield"
```
Comment (XSlayer300) [inline code/game/objects/items/weapons/shields.dm:129]: im a simple man
i see multiple upper case in first letters
i clicc
```suggestion
	name = "\improper TL-172 defensive shield"
```

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-2). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**